poolmgr/gp.go

```diff
@@ -487,6 +514,23 @@ func (gp *GenericPool) CleanupFunctionService(podName string) error {
 		return nil
 	}
 
+	pod, err := gp.kubernetesClient.Core().Pods(gp.namespace).Get(podName)
+	if err != nil {
+		return err
+	}
+
+	loggerUrl := fmt.Sprintf("http://%s:1234/v1/log/%s", pod.Spec.NodeName, pod.Name)
```

Comment:
I'll add the env support in later PR with changes that cli retrieves log stream from controller instead of connecting to database directly.